package.json

```diff
@@ -55,11 +55,13 @@
     "@ethersproject/networks": "^5.0.4",
     "@ethersproject/providers": "^5.0.9",
     "@ethersproject/solidity": "^5.0.4",
-    "@material-ui/core": "5.0.0-alpha.21",
+    "@material-ui/core": "^5.0.0-alpha.21",
```

Comment:
why?

Reply:
@septs My Npm wasn't recognising the material ui package, so I used the Npm upgrade